setup.py

```diff
@@ -90,6 +90,12 @@ def run(self):
             targetFileName = os.path.join(dir, baseName)
             self.copy_file(command.importLibraryName, targetFileName)
             self.outfiles.append(targetFileName)
+            # install cx_Logging.h
```

Comment:
Is this only relevant on Windows?

Reply:
I think so because for other systems the library isn't distributed, then the user will use the source.

Reply:
Ok. Can you merge with the latest changes on master and adjust the comment on line 79-80 as well? Thanks.

Reply:
done